lightning/src/ln/interactivetxs.rs

```diff
@@ -418,6 +476,13 @@ impl InteractiveTxSigningSession {
 	}
 }
 
+impl_writeable_tlv_based!(InteractiveTxSigningSession, {
+	(1, unsigned_tx, required),
+	(3, holder_sends_tx_signatures_first, required),
+	(5, has_received_commitment_signed, required),
+	(7, holder_tx_signatures, required),
+});
```

Comment:
IIUC, when introducing a new serialization, we can number these starting at `0` and use both even and odd given no previous versions understand this. Likewise elsewhere.

Reply:
Actually, given @wpaulino [comment](https://github.com/lightningdevkit/rust-lightning/pull/3651#discussion_r2011026712) on another PR, I'm not sure if this would be desirable. @TheBlueMatt What are the best practices when adding serialization for a new struct? IIRC, you had once suggested just numbering with both even and add. But Wilmer's comment of only using odd makes sense to me.